static/app/views/explore/logs/logsTab.tsx

```diff
@@ -250,24 +250,26 @@ export function LogsTabContent({
               />
             </StyledPageFilterBar>
             <TraceItemSearchQueryBuilder {...tracesItemSearchQueryBuilderProps} />
-            <DropdownMenu
-              items={saveAsItems}
-              trigger={triggerProps => (
-                <Button
-                  {...triggerProps}
-                  priority="primary"
-                  aria-label={t('Save as')}
-                  onClick={e => {
-                    e.stopPropagation();
-                    e.preventDefault();
-
-                    triggerProps.onClick?.(e);
-                  }}
-                >
-                  {t('Save as')}
-                </Button>
-              )}
-            />
+            {saveAsItems.length > 0 && (
```

Comment:
We can just get rid of the dashboard flag, this case should never happen as dashboards at least are fully released to open beta